engine/userguide/networking/default_network/ipv6.md

```diff
@@ -92,11 +92,11 @@ on the host:
 
 ![](images/ipv6_slash64_subnet_config.svg)
 
-In this setup the subnet `2001:db8:23:42::/80` with a range from
-`2001:db8:23:42:0:0:0:0` to `2001:db8:23:42:0:ffff:ffff:ffff` is attached to
+In this setup the subnet `2001:db8:23:42::/64` with a range from
+`2001:db8:23:42:0:0:0:0` to `2001:db8:23:42:ffff:ffff:ffff:ffff` is attached to
 `eth0`, with the host listening at `2001:db8:23:42::1`. The subnet
 `2001:db8:23:42:1::/80` with an address range from `2001:db8:23:42:1:0:0:0` to
-`2001:db8:23:42:1:ffff:ffff:ffff` is attached to `docker0` and will be used by
+`2001:db8:23:42:ffff:ffff:ffff:ffff` is attached to `docker0` and will be used by
```

Comment:
Please restore this line, this was correct. `\80` means the first 80 bits do not change, so the first bit groups (16 bits each) in the address do not change: `2001:db8:23:42:1:`